src/amo/components/SearchFilters/index.js

```diff
@@ -112,6 +122,33 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
     return false;
   };
 
+  changeColorFilter: (colorValue: string) => void = (colorValue: string) => {
+    const { filters } = this.props;
+    const newFilters = { ...filters };
+    if (filters.color === colorValue) {
+      delete newFilters.color;
+    } else if (colorValue) {
+      newFilters.color = colorValue;
+    }
+
+    this.doSearch(newFilters);
+  };
+
+  onColorClick: HTMLElementEventHandler = (event: ElementEvent) => {
+    event.preventDefault();
+    const colorValue = event.currentTarget.getAttribute('data-color') || '';
+    this.changeColorFilter(colorValue);
+  };
+
+  onCustomColorChange: (event: TypedElementEvent<HTMLInputElement>) => void = (
```

Comment:
I think I want to challenge the utility of having a custom color selection... is someone really going to search for themes that have #FF2253 .... I don't buy it as being worth the extra code, testing, etc.

Reply:
That was the first thing people I showed the prototype to at MozWeek asked for (I initially didn't have it). I wasn't convinced at first, but it wasn't too hard to implement, and as mentioned in the description, gives us the ability to search for colors like pink, orange, etc.